src/Core/DynamicMetadata/MultiAreas.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Rokka\Client\Core\DynamicMetadata;
+
+use Rokka\Client\DynamicMetadataHelper;
+
+class MultiAreas implements DynamicMetadataInterface
+{
+    /**
+     * @var array
+     */
+    private $areas = [];
+
+    /**
+     * @param array $areas
+     */
+    public function __construct($areas)
+    {
+        $this->areas = $areas;
+    }
+
+    /**
+     * @return string The name of the metadata
+     */
+    public static function getName()
+    {
+        return 'multi_areas';
+    }
+
+    /**
+     * Get all Areas.
+     *
+     * @return array
+     */
+    public function getAreas()
+    {
+        return $this->areas;
+    }
+
+    /**
+     * Gets an array of Areas with a specific name.
+     *
+     * @param string $name
+     *
+     * @return DynamicMetadataInterface[]|null
+     */
+    public function getArea($name)
```

Comment:
I am a bit confused by "Gets an array of Areas with a specific name." and then the function being called "getArea", I know there is a getAreas, but maybe "getAreasForName"?